BlazorHero.CleanArchitecture/Server/Permission/PermissionAuthorizationHandler.cs

```diff
@@ -0,0 +1,36 @@
+﻿using BlazorHero.CleanArchitecture.Shared.Constants.Permission;
+using BlazorHero.CleanArchitecture.Shared.Models.Identity;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace BlazorHero.CleanArchitecture.Server.Permission
+{
+    internal class PermissionAuthorizationHandler : AuthorizationHandler<PermissionRequirement>
+    {
+
+        public PermissionAuthorizationHandler()
+        { }
+        protected override async Task HandleRequirementAsync(AuthorizationHandlerContext context, PermissionRequirement requirement)
+        {
+
+            if (context.User == null) // roleclaims is not accessible - TODO: 
```

Comment:
I was going to implement for blazor  permission authorization as well but could not undertand clearly why the role claims were not accessible here (empty)